ui/app/AppLayouts/Profile/views/EnsDetailsView.qml

```diff
@@ -60,17 +60,20 @@ Item {
             walletAddressLbl.visible = true;
             keyLbl.visible = true;
             releaseBtn.visible = isStatus
+            removeButton.visible = true
             releaseBtn.enabled = (Date.now() / 1000) > expirationTime && expirationTime > 0 &&
                     root.ensUsernamesStore.preferredUsername != username
-                    releaseBtn.enabled = true
+            releaseBtn.enabled = true
```

Comment:
Fishy at least... why would you set `releaseBtn.enabled` to something and then on the next line to `true` unconditionally?

Reply:
Oops that's something that should be removed.  I had added `releaseBtn.enabled = true` while i was developing #6501 so I could have that button enabled always without waiting for the expiration time, but forgot to remove that line.

Reply:
FIXIT :smiley: